pkg/operation/shoot/types.go

```diff
@@ -32,5 +32,6 @@ type Shoot struct {
 	KubernetesMajorMinorVersion string
 
 	WantsClusterAutoscaler bool
+	WantsAlertmanager      bool
```

Comment:
I'd not add this field here but rather give the health check this single boolean.